pulumi.yaml

```diff
@@ -0,0 +1,79 @@
+package:
+  name: pulumi
+  version: 3.65.1
+  epoch: 0
+  description: Infrastructure as Code in any programming language
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - go
+      - busybox
+      - build-base
+      - yarn
+      - nodejs
+      - python3
+      - python3-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/pulumi/pulumi.git
+      tag: v${{package.version}}
+      destination: ${{package.name}}
+      expected-commit: f4044dc955009a7da3d58f845ec29a9a7f80ec49
+
+  - working-directory: ${{package.name}}
+    pipeline:
+      - runs: |
+          set -x
+          export CGO_ENABLED=0 GO111MODULE=on
+          export PULUMI_VERSION="v${{package.version}}"
+          export PULUMI_ROOT="$(mktemp -d)"
+          export GOBIN="${PULUMI_ROOT}/bin"
+
+          # Build the CLI itself
+          make install
+
+          # Build the Pulumi Language SDKs (Go, Node.js, Python)
+          for lang in go nodejs python; do
+            (cd sdk/${lang} && make build)
+          done
```

Comment:
it doesnt appear to be. besides, these are also being split into subpackages with hardcoded names below. If we allow new SDKs in without knowing about them, new binaries will appear in the main package when we want it in a subpackage. This would also likely require more environment packages